README.md

```diff
@@ -51,3 +51,35 @@ Code coverage will be generated on path:
 ```
 {project-root}/{module}/target/scala-{scala_version}/jacoco/report/html
 ```
+## Health check endpoint
+Springboot Actuator enabled for this project. This provides the user with an endpoint (readable via http or JMX)
+that describes the status of the login-gateway as well as it's dependencies (for example: Databases, Ldap, server statistics, etc).
```

Comment:
```suggestion
that describes the overall status of the login-gateway as well as its parts.
```
(I would leave out the examples, because they are not true currently)